detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>[line 66] in the print statement, you use "detector.ap". does that work?</p><p>[line 41] probably should use os.path.join instead</p><p>[minor] maybe we should run this cell through the black formatter?</p><p> </p><p>cool to see that the DetectionDataset and DetectionLeaner as nice and easy to use!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
- line 66: yes, it works. Except that sometimes the values are negative but should be 0. Typically after training a while the values match exactly the once printed out by torchvision after each epoch
- line 41: agreed.
- as for black formatter: for the notebooks I feel that sometimes not using black is more human friendly. E.g. takes up less lines and hence makes it easier to read. For the notebooks I am therefore personally less worried about code consistency.

Reply:
And yes, DetectionDataset and DetectionLearner are super simple to use!!! @jiata